packages/airnode-protocol/test/utils.js

```diff
@@ -7,7 +7,7 @@ function deriveWalletPathFromSponsorAddress(sponsorAddress) {
     const shiftedSponsorAddressBN = sponsorAddressBN.shr(31 * i);
     paths.push(shiftedSponsorAddressBN.mask(31).toString());
   }
-  return `1/${paths.join('/')}`;
+  return `0/${paths.join('/')}`;
```

Comment:
Changes like this will have to be made in a number of places, I'm just confirming that reverting those changes will be a separate task?
I count that exact path string in at least 11 places in airnode.

Reply:
This one should actually not be changed (it should be 1)

Reply:
Oops, good find